srml/contracts/src/exec.rs

```diff
@@ -254,13 +254,28 @@ impl<T: Trait> Token<T> for ExecFeeToken {
 	}
 }
 
+#[cfg_attr(any(feature = "std", test), derive(Debug, PartialEq, Eq, Clone))]
```

Comment:
Hehe, grep for `26925` in the codebase.

Here the bound for derived traits will be (like, `T: Debug`) which doesn't make sense for configuration traits, because what really matters is that used types have these bounds (like, `T::AccountId: Debug`)